src/linux.rs

```diff
@@ -3,24 +3,26 @@
 use crate::Error;
 use async_trait::async_trait;
 use futures::{future::ready, Stream, StreamExt};
-use std::{fs, io, path::PathBuf};
+use std::{io, path::PathBuf};
+use tokio::fs;
```

Comment:
this is the main change to this file - switching from blocking `std::fs` to async `tokio::fs`